[#227] download IEDriverServer from SeleniumHQ/selenium GitHub releases

aleksandr-kotlyar opened the pull request:
- adopt IEDriverServer to be downloaded from SeleniumHQ/selenium GitHub repo release assets instead of https://selenium-release.storage.googleapis.com/
- format indentations and linebreaks in GeckoDriver and IEDriver classes

Closes #227 
Closes #217 
Closes #216
Closes #221

pep8speaks commented:
Hello @aleksandr-kotlyar! Thanks for opening this PR. We checked the lines you've touched for [PEP 8](https://www.python.org/dev/peps/pep-0008) issues, and found:



* In the file [`webdriver_manager/driver.py`](https://github.com/SergeyPirogov/webdriver_manager/blob/12a2434286b3023b82f330941caa6610f591f46d/webdriver_manager/driver.py):

> [Line 107:80](https://github.com/SergeyPirogov/webdriver_manager/blob/12a2434286b3023b82f330941caa6610f591f46d/webdriver_manager/driver.py#L107): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (118 > 79 characters)
> [Line 178:80](https://github.com/SergeyPirogov/webdriver_manager/blob/12a2434286b3023b82f330941caa6610f591f46d/webdriver_manager/driver.py#L178): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (118 > 79 characters)

* In the file [`webdriver_manager/microsoft.py`](https://github.com/SergeyPirogov/webdriver_manager/blob/12a2434286b3023b82f330941caa6610f591f46d/webdriver_manager/microsoft.py):

> [Line 12:80](https://github.com/SergeyPirogov/webdriver_manager/blob/12a2434286b3023b82f330941caa6610f591f46d/webdriver_manager/microsoft.py#L12): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (80 > 79 characters)
> [Line 13:80](https://github.com/SergeyPirogov/webdriver_manager/blob/12a2434286b3023b82f330941caa6610f591f46d/webdriver_manager/microsoft.py#L13): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (103 > 79 characters)
> [Line 14:80](https://github.com/SergeyPirogov/webdriver_manager/blob/12a2434286b3023b82f330941caa6610f591f46d/webdriver_manager/microsoft.py#L14): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (110 > 79 characters)

aleksandr-kotlyar commented:
I am not sure that line length of these counts.
First two are docstrings.
Second two are string variables.
If I need to fix them too - please tell me.

> Hello @aleksandr-kotlyar! Thanks for updating this PR. We checked the lines you've touched for [PEP 8](https://www.python.org/dev/peps/pep-0008) issues, and found:
> 
> * In the file [`webdriver_manager/driver.py`](https://github.com/SergeyPirogov/webdriver_manager/blob/5e59a81e9d0f6824b4d3e34a71c71371ed505580/webdriver_manager/driver.py):
> 
> > [Line 107:80](https://github.com/SergeyPirogov/webdriver_manager/blob/5e59a81e9d0f6824b4d3e34a71c71371ed505580/webdriver_manager/driver.py#L107): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (118 > 79 characters)
> > [Line 178:80](https://github.com/SergeyPirogov/webdriver_manager/blob/5e59a81e9d0f6824b4d3e34a71c71371ed505580/webdriver_manager/driver.py#L178): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (118 > 79 characters)
> 
> * In the file [`webdriver_manager/microsoft.py`](https://github.com/SergeyPirogov/webdriver_manager/blob/5e59a81e9d0f6824b4d3e34a71c71371ed505580/webdriver_manager/microsoft.py):
> 
> > [Line 15:80](https://github.com/SergeyPirogov/webdriver_manager/blob/5e59a81e9d0f6824b4d3e34a71c71371ed505580/webdriver_manager/microsoft.py#L15): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (94 > 79 characters)
> > [Line 16:80](https://github.com/SergeyPirogov/webdriver_manager/blob/5e59a81e9d0f6824b4d3e34a71c71371ed505580/webdriver_manager/microsoft.py#L16): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (101 > 79 characters)
> 
> ##### Comment last updated at 2021-10-06 00:06:33 UTC

aleksandr-kotlyar commented:
Opera Tests fails cause latest opera webdriver is newer than browser version installs in CI. So it doesn't count for this PR.

@SergeyPirogov please take a review. I have fixed IEDriver Downloads by changing IEDriver repo.

aleksandr-kotlyar commented:
> @aleksandr-kotlyar tests are failing
@SergeyPirogov read the message below please :)  

> Opera Tests fails cause latest opera webdriver is newer than browser version installs in CI. So it doesn't count for this PR.
> 
> @SergeyPirogov please take a review. I have fixed IEDriver Downloads by changing IEDriver repo.

I know. And they fail cause of another error.

Here I changed only code for IEDriver.
Opera test fails - because latest webdriver is higher version than ubuntu/macos repositories have opera browser. And it will be fixed when webdriver_manager will implement this feature #233. I have already started work on it. But this is not about this PR.
